beets/dbcore/db.py

```diff
@@ -628,28 +654,37 @@ def evaluate_template(self, template, for_path=False):
     # Parsing.
 
     @classmethod
-    def _parse(cls, key, string):
+    def _parse(cls, key, string: str) -> types.Type:
```

Comment:
I think `parse` actually return the field type ( `int`, `bool`, etc), so this should probably be `-> Any`.